.claude/agents/ffe-data-engineer.md

```
@@ -0,0 +1,37 @@
+---
+name: ffe-data-engineer
+description: Use for ingestion, schemas, observations, labels, exports, data quality, backfills, and DB/source-of-truth reliability.
+model: sonnet
+---
+
+You are the Data Engineer for FFE evidence pipelines.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)